src/AzureAuth/CommandMain.cs

```diff
@@ -376,14 +379,20 @@ private IAuthFlow AuthFlow()
                 // TODO: Really we need to get rid of Resource
                 var scopes = this.Scopes ?? new string[] { $"{this.authSettings.Resource}/.default" };
 
+                string promptHint = PromptHintPrefix;
+                if (string.IsNullOrEmpty(promptHint))
+                {
+                    promptHint = $"{PromptHintPrefix}: {promptHint}";
+                }
+
                 var authFlows = AuthFlowFactory.Create(
                     this.logger,
                     this.CombinedAuthMode,
                     new Guid(this.authSettings.Client),
                     new Guid(this.authSettings.Tenant),
                     scopes,
                     this.PreferredDomain,
-                    this.authSettings.PromptHint,
+                    promptHint,

```

Comment:
Do we lose the user-configured prompt hint this way?

Reply:
Same mistake with the upper conversation.